requirements/constraints.txt

```diff
@@ -40,9 +40,6 @@ edx-enterprise==3.10.3
 # v2 requires the ES7 upgrade work to be complete
 edx-search<2.0.0
 
-# We expect v2.0.0 to introduce large breaking changes in the feature toggle API
-edx-toggles<2.0.0
-
```

Comment:
Let's leave this until we actually switch to 2.0.0, and update this PR to work off the new version.

Reply:
I don't quite understand this comment (see my question on Slack).